bcipy/gui/params_form.py

```diff
@@ -67,22 +65,37 @@ def createControls(self):
         parameters file."""
 
         self.controls = {}
-        for key, param in self.params.items():
-            if param.type == 'bool':
-                form_input = self.bool_input(param)
-            elif 'path' in param.type:
-                form_input = self.file_input(param)
-            elif isinstance(param.recommended_values, list):
-                form_input = self.selection_input(param)
-            # TODO: NumCtrl for numeric input types.
-            # from wx.lib.masked import NumCtrl
-            # elif param['type'] in ['float', 'int']:
+        # Retrieve and alphebatize list of unique parameter sections
+        param_sections = sorted(list(set([param[1].section for param in self.params.items()])))
+        # Sort parameters by section
+        key_dict = []
+        for each_section in param_sections:
+            # Get readable names for sections
+            name_dict = ast.literal_eval(self.params['section_names'].value)
```

Comment:
This line throws a KeyError when section_names is missing. I would prefer to have this configuration elsewhere and not require an eval.